### PR TITLE
Merge F-10B code into main.

### PR DESCRIPTION
**Pull request by ajaysrivas**
Merge F-10B code into main.

### CODE REVIEW

File: ha/const.py [hunk 1]
```diff
@@ -52,10 +53,31 @@
 SOURCE_CLI_SCHEMA_FILE = "{}/cli_schema.json".format(SOURCE_CONFIG_PATH)
 CLI_SCHEMA_FILE = "{}/cli_schema.json".format(CONFIG_DIR)
 COMPONENTS_CONFIG_DIR = "{}/components".format(CONFIG_DIR)
+SOURCE_HEALTH_HIERARCHY_FILE = "{}/system_health_hierarchy.json".format(SOURCE_CONFIG_PATH)
+HEALTH_HIERARCHY_FILE = "{}/system_health_hierarchy.json".format(CONFIG_DIR)
+IEM_SCHEMA="{}/iem_ha.json".format(CONFIG_DIR)
+
+# IEM DESCRIPTION string: To be removed
```
Comment (indrajitzagade) [inline ha/const.py:60]: This is repeated four times

File: ha/const.py [hunk 2]
```diff
@@ -148,14 +170,28 @@
...
 SYSTEM_SERVICE_FILE = "{}/event_analyzer.service".format(SYSTEM_DIR)
 SOURCE_SERVICE_FILE = "{}/conf/service/event_analyzer.service".format(SOURCE_PATH)
+ACTUATOR_RESPONSE_TYPE= "actuator_response_type"
+SENSOR_RESPONSE_TYPE= "sensor_response_type"
+SPECIFIC_INFO = "specific_info"
+INFO = "info"
+COMPONENT = "component"
+MODULE = "module"
+RESOURCE_TYPE = "resource_type"
+IEM_DESCRIPTION="WS0080010001, Node, The cluster has lost $host server. System is running in degraded mode. " \
```
Comment (indrajitzagade) [inline ha/const.py:187]: It is here as well

File: ha/const.py (hunk 1 above)
Comment (ajaysrivas) [reply ha/const.py:60]: Loosing confidence

File: ha/const.py (hunk 2 above)
Comment (ajaysrivas) [reply ha/const.py:178]: Nothing here. What is it?

Comment (indrajitzagade) [reply ha/const.py:178]: Same string as line #61

Comment (ajaysrivas) [reply ha/const.py]: done.

**Review by indrajitzagade — APPROVED** [review]

**Review by ajay-paratmandali — APPROVED** [review]

**Review by ArchanaLimaye — APPROVED** [review]

**Pull request merged**